src/Localization/en/translations.json

```diff
@@ -37,6 +37,56 @@
             "OnSaveFailureMessage": "Data save operation not currently permitted."
         },
         "SqlX": {
+            "DedicatedGatewayDescription": "Provision a dedicated gateway cluster for your Azure Cosmos DB account. A dedicated gateway is compute that is a front-end to data in your Azure Cosmos DB account. Your dedicated gateway automatically includes the integrated cache, which can improve read performance. ",
+            "DedicatedGateway": "Dedicated Gateway",
+            "Enable": "Enable",
+            "Disable": "Disable",
+            "LearnAboutDedicatedGateway": "Learn more about dedicated gateway.",
+            "DeprovisioningDetailsText": "Learn more about deprovisioning the dedicated gateway.",
+            "DedicatedGatewayPricing": "Learn more about dedicated gateway pricing",
```

Comment:
this shows up in both the warning banner and in the cost description. Seems like redundant info. DO we need it?

Reply:
I think it is okay. Eventually, in the mid-term, we will replace cost description with the actual estimate. Also, this was approved by Tim and we should be okay with it. Changing this is fairly straight-forward